packages/fetchai/skills/tac_participation/skill.yaml

```diff
@@ -29,6 +31,10 @@ models:
     class_name: Search
     args:
       search_interval: 5
+  parameters:
+    class_name: Parameters
+    args:
+      is_using_contract: False
```

Comment:
We can specify if we are using the contract or not in the skill config. We will need to add this also in the negotiation skill or save it in the `shared state`.